app/controllers/moirai/translation_files_controller.rb

```diff
@@ -60,6 +70,15 @@ def handle_create
         return
       end
 
+      if translation_params[:value].blank? && request.format.json?
+        respond_to do |format|
+          format.json do
+            render json: {fallback_translation: get_fallback_translation}
+          end
+        end
+        return
+      end
```

Comment:
No need for respond_to if the format is already being checked.
```suggestion
      if translation_params[:value].blank? && request.format.json?
        return render json: {fallback_translation: get_fallback_translation}
      end
```